box/box.go

```diff
@@ -17,6 +17,12 @@ func New(conn tarantool.Doer) *Box {
 	}
 }
 
+// Schema returns a new Schema instance, providing access to schema-related operations.
+// It uses the connection from the Box instance to communicate with Tarantool.
+func (b *Box) Schema() *Schema {
+	return NewSchema(b.conn)
+}
```

Comment:
It would be nice to write a test that checks whether the created circuit uses the connection passed to `box.New`.